doc/md/tutorial-todo-gql.md

```diff
@@ -0,0 +1,349 @@
+	if err != nil {
+		log.Fatal("opening ent client", err)
+	}
+	if err := client.Schema.Create(
+		context.Background(),
+		migrate.WithGlobalUniqueID(true),
+	); err != nil {
+		log.Fatal("opening ent client", err)
+	}
+
+	// Configure the server and start listening on :8081.
+	srv := handler.NewDefaultServer(NewSchema(client))
+	http.Handle("/",
+		playground.Handler("Todo", "/query"),
+	)
+	http.Handle("/query", srv)
+	log.Println("listening on :8081")
+	if err := http.ListenAndServe(":8081", nil); err != nil {
+		log.Fatal("http server terminated", err)
+	}
+}
+
+```
+
+Run the server using the command below, and open [localhost:8081](http://localhost:8081):
+
+```console
+go run ./cmd/todo
+```
+
+You should see the interactive playground:
+
+![tutorial-todo-playground](https://entgo.io/assets/tutorial-gql-playground.png)
+
+If you're having troubles with getting the playground, go to [first section](#clone-the-code-optional) and clone the
+example repository.
+
+## Query Todos
+
+If we'll try to query our todo list, we'll get an error as the resolver method was not implemented. 
+Let's implement the resolver by replacing the `Todos` implementation in the query resolver:
+
+```diff
+func (r *queryResolver) Todos(ctx context.Context) ([]*ent.Todo, error) {
+-	panic(fmt.Errorf("not implemented"))
++	return r.client.Todo.Query().All(ctx)
+}
+```
+
+Then, running this GraphQL query should return an empty todo list:
+
+```graphql
+query AllTodos {
+    todos {
+        id
+    }
+}
+
+# Output: { "data": { "todos": [] } }
+```
+
+## Create a Todo
+
+Same as before, if we'll try to create a todo item in GraphQL we'll get an error as the resolver was not implemented.
+Let's implement the resolver by changing the `CreateTodo` implementation in the mutation resolver:
+
+```go
+func (r *mutationResolver) CreateTodo(ctx context.Context, todo TodoInput) (*ent.Todo, error) {
+	return r.client.Todo.Create().
+		SetText(todo.Text).
+		SetStatus(todo.Status).
+		SetNillablePriority(todo.Priority). // Set the "priority" field if provided.
+		SetNillableParentID(todo.Parent).   // Set the "parent_id" field if provided.
+		Save(ctx)
+}
+```
+
+Then, creating a todo item should work:
+
+```graphql
+mutation CreateTodo($todo: TodoInput!) {
+    createTodo(todo: $todo) {
+        id
+        text
+        createdAt
+        priority
+        parent {
+            id
+        }
+    }
+}
+
+# Query Variables: { "todo": { "text": "Create GraphQL Example", "status": "IN_PROGRESS", "priority": 1 } }
+# Output: { "data": { "createTodo": { "id": "2", "text": "Create GraphQL Example", "createdAt": "2021-03-10T15:02:18+02:00", "priority": 1, "parent": null } } }
+```
+
+If you're having troubles with getting this example to work, go to [first section](#clone-the-code-optional) and clone the
+example repository.
+
+---
+
+Please continue to the next section and learn how to Relay Node Interface with Ent.
```

Comment:
maybe how to use/integrate/configure Relay Node Interface with Ent. (A sentence about it here could be useful as a teaser on what you will learn in the next phase)